.goreleaser.yaml

```diff
@@ -126,69 +125,45 @@ universal_binaries:
     ids:
       - osmosisd-darwin-amd64
       - osmosisd-darwin-arm64
-    replace: true
-    name_template: "{{.ProjectName}}"
+    replace: false
 
 archives:
   - id: zipped
     builds:
       - osmosisd-darwin-universal
       - osmosisd-linux-amd64
       - osmosisd-linux-arm64
-    name_template: "{{.ProjectName}}-{{ .Os }}-{{ .Arch }}"
+      - osmosisd-darwin-amd64
+      - osmosisd-darwin-arm64
+    name_template: "{{.ProjectName}}-{{ .Version }}-{{ .Os }}-{{ .Arch }}"
     format: tar.gz
     files:
       - none*
   - id: binaries
     builds:
       - osmosisd-darwin-universal
       - osmosisd-linux-amd64
-      - osmosisd-linux-arm64
-    name_template: "{{.ProjectName}}-{{ .Os }}-{{ .Arch }}"
+      - osmosisd-linux-amd64
+      - osmosisd-darwin-amd64
+      - osmosisd-darwin-arm64
+    name_template: "{{.ProjectName}}-{{ .Version }}-{{ .Os }}-{{ .Arch }}"
     format: binary
     files:
       - none*
 
 checksum:
-  name_template: "checksums.txt"
+  name_template: "sha256sum.txt"
   algorithm: sha256
 
-# Docs: https://goreleaser.com/customization/homebrew/
-brews:
-  - name: osmosisd
-    folder: Formula
-    ids:
-    - binaries
-    homepage: "https://gihub.com/osmosis-labs/osmosis"
-    description: "osmosisd binary to interact with the Osmosis network"
-    conflicts:
-      - osmosisd
-    test: |
-      system "#{bin}/osmosisd version"
-    install: |
-      bin.install 'osmosisd'
-    skip_upload: true
-    # # Uncomment line below if you want to try to commit the updated formula
-    # skip_upload: false
-    # repository:
-    #   owner: osmosis-labs
-    #   name: homebrew-osmosis
-    #   branch: main
-    #   pull_request:
-    #     enabled: true
-    # commit_author:
-    #   name: osmo-bot
-    #   email: devops@osmosis.team
-    # commit_msg_template: "Brew formula update for {{ .ProjectName }} version {{ .Tag }}"
-
 # Docs: https://goreleaser.com/customization/changelog/
 changelog:
   skip: true
 
 # Docs: https://goreleaser.com/customization/release/
 release:
   github:
-    owner: osmosis-labs
+    owner: niccoloraspa
+    # owner: osmosis-labs
```

Comment:
intentional?

Reply:
No it's a mistake. I changed it for testing.

Thanks for spotting it